designer/server/src/config.ts

```diff
@@ -89,31 +118,32 @@ const schema = joi.object<Config>({
 // Validate config
 const result = schema.validate(
   {
-    port: process.env.PORT,
-    env: process.env.NODE_ENV,
-    managerUrl: process.env.MANAGER_URL,
-    submissionUrl: process.env.SUBMISSION_URL,
-    previewUrl: process.env.PREVIEW_URL,
+    port: PORT,
+    env: DEPLOYMENT_ENV,
+    nodeEnv: NODE_ENV,
+    managerUrl: MANAGER_URL,
+    submissionUrl: SUBMISSION_URL,
+    previewUrl: PREVIEW_URL,
     serviceName: 'Submit a form to Defra',
-    logLevel: process.env.LOG_LEVEL,
-    phase: process.env.PHASE,
-    isProduction: process.env.NODE_ENV === 'production',
-    isDevelopment: !['production', 'test'].includes(`${process.env.NODE_ENV}`),
-    isTest: process.env.NODE_ENV === 'test',
-    sessionTtl: process.env.SESSION_TTL,
-    fileDownloadPasswordTtl: process.env.FILE_DOWNLOAD_PASSWORD_TTL,
-    sessionCookiePassword: process.env.SESSION_COOKIE_PASSWORD,
-    sessionCookieTtl: process.env.SESSION_COOKIE_TTL,
-    azureClientId: process.env.AZURE_CLIENT_ID,
-    azureClientSecret: process.env.AZURE_CLIENT_SECRET,
-    oidcWellKnownConfigurationUrl:
-      process.env.OIDC_WELL_KNOWN_CONFIGURATION_URL,
-    appBaseUrl: process.env.APP_BASE_URL,
-    redisHost: process.env.REDIS_HOST,
-    redisUsername: process.env.REDIS_USERNAME,
-    redisPassword: process.env.REDIS_PASSWORD,
-    redisKeyPrefix: process.env.REDIS_KEY_PREFIX,
-    roleEditorGroupId: process.env.ROLE_EDITOR_GROUP_ID
+    logLevel: LOG_LEVEL,
+    phase: PHASE,
+    isProduction: DEPLOYMENT_ENV === 'prod',
+    isDevelopment:
+      !DEPLOYMENT_ENV || !['prod', 'test'].includes(DEPLOYMENT_ENV),
```

Comment:
Is this still what we want?

Reply:
Yeah think so

CDP do have the description "**If this application running in the development environment**" but they similarly set it to `process.env.NODE_ENV !== 'production'` so I'm happy with your changes

i.e. You've kept the same behaviour yet we can adjust it in future if needed